packages/electron-host/src/start-electron-env.ts

```diff
@@ -107,22 +107,5 @@ export async function runElectronEnv({
         env,
     });
 
-    // eslint-disable-next-line @typescript-eslint/no-misused-promises
-    app.on('window-all-closed', async () => {
-        // Don't quit between closing one window and immediately opening another.
-        await new Promise((res) => setTimeout(res, 0));
-
-        if (BrowserWindow.getAllWindows().length === 0) {
-            try {
-                await runningEnvironment.dispose();
-            } catch (e) {
-                process.exitCode = 1;
-                // eslint-disable-next-line no-console
-                console.error(e);
-            }
-            app.quit();
```

Comment:
how does quite event triggered now?

Reply:
@olehrakwix if `window-all-closed` listener has no subscribers the default behavior would be to quit the app, thus trigger the event (according to [doc](https://www.electronjs.org/docs/latest/api/app#event-window-all-closed))